### ISSUE TITLE
Possible RAC integration bug

### ISSUE THREAD

**Issue by kzaher** [Moya/Moya#165]
Hi Ash,

I've seen a possible bug in Moya RAC integration, but I'm not a RAC expert so not sure if it's really a bug.

I think that [here](https://github.com/ashfurrow/Moya/blob/master/Moya/ReactiveCocoa/Moya%2BReactiveCocoa.swift#L52) it should be
```
.replayLast().autoconnect()
```

instead of
```
.publish().autoconnect()
```

but I could be wrong.

**Comment by ashfurrow** [comment]
I did this originally to implement inflight request tracking; a multicast connection's `autoconnect()` generates a new signal every time (subscribed to the same underlying signal), so I couldn't store it in the requests dictionary. I got a little frustrated and settled with a solution that appeared to work. I should review the docs and implementation. Since Moya has good test coverage, we can be sure that any changes don't break behaviour :smile:
